build.gradle

```diff
@@ -197,10 +199,11 @@ task shadowTestJar(type: ShadowJar) {
         include(dependency('org.json4s:json4s-jackson_2.10:.*'))
         include(dependency('org.json4s:json4s-ast_2.10:.*'))
 
-        include(dependency('org.testng:testng:.*'))
-        include(dependency('com.beust:jcommander:.*'))
-        include(dependency('org.scalatest:scalatest_' + scalaMajorVersion + ':.*'))
-
+        include(dependency('org.http4s:http4s-core_2.10:.*'))
+        include(dependency('org.scalaz:scalaz-core_2.10:.*'))
+        include(dependency('org.scalaz:scalaz-concurrent_2.10:.*'))
+        include(dependency('org.scalaz:scalaz-stream_2.10:.*'))
```

Comment:
ooo `scalaz`! 😁

We should probably use the `scalaMajorVersion` variable here, right?